spec/api/v1/schemas/benchmarks.rb

```diff
@@ -27,6 +27,9 @@ module Benchmarks
             },
             description: {
               type: 'string'
+            },
+            os_major_version: {
+              type: 'string'
```

Comment:
We might reconsider using an integer, for ease of comparison.

Reply:
Yeah, it's not a bad idea, but this info comes from the inventory API which returns the version info as a string; it's not always of the format `"7.6"`, so for now I think it makes sense to match their API and return a string. It also should match the OS info we give for other resources, such as [profiles](https://github.com/RedHatInsights/compliance-backend/blob/master/spec/api/v1/schemas/profiles.rb#L73).